<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Swap Courses</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="swap_courses_tableview">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>811</width>
      <height>201</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Course ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Course Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Section</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Instructor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Days</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>End Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Room</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="swap_courses_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Swap Courses</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_button">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>310</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="swap_enrolled_courses_tableview">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>811</width>
      <height>211</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Course ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Course Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Section</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Instructor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Days</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>End Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Room</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="swap_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>640</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Swap</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
